### PR TITLE
Changelog for v2.14.0

### PR DESCRIPTION
**Pull request by kdomanski**
```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,108 @@
+### [v2.14.0]
+
+
+**Misc:**
+
+- ACTION REQUIRED: The most recent backup for user clusters is kept when the cluster is deleted. Adjust the cleanup-container to get the old behaviour (delete all backups) back. [#5262](https://github.com/kubermatic/kubermatic/issues/5262) ([xrstf](https://github.com/xrs
+tf))
```
Comment (xrstf) [inline CHANGELOG.md]: bad linebreak

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,108 @@
...
+- ACTION REQUIRED: Addon manifest templating is now a stable API, but different to the old implicit data. Custom addons might need to be adjusted. [#5275](https://github.com/kubermatic/kubermatic/issues/5275) ([xrstf](https://github.com/xrstf))
+- Added Flatcar Linux as an Operating System option
+- Add SLES as an Operating System option [#5040](https://github.com/kubermatic/kubermatic/issues/5040) ([kgroschoff](https://github.com/kgroschoff))
+- Audit logging can now be enforced in all clusters within a Datacenter. [#5045](https://github.com/kubermatic/kubermatic/issues/5045) ([kdomanski](https://github.com/kdomanski))
+- Added Kubernetes v1.15.10, v1.16.7, v1.17.3 [#5102](https://github.com/kubermatic/kubermatic/issues/5102) ([kdomanski](https://github.com/kdomanski))
+- Add support for Kubernetes 1.18, drop support for Kubernetes &lt; 1.15. [#5325](https://github.com/kubermatic/kubermatic/issues/5325) ([xrstf](https://github.com/xrstf))
```
Comment (xrstf) [inline CHANGELOG.md]: Please merge these two entries.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,108 @@
+### [v2.14.0]
+
+
+**Misc:**
+
+- ACTION REQUIRED: The most recent backup for user clusters is kept when the cluster is deleted. Adjust the cleanup-container to get the old behaviour (delete all backups) back. [#5262](https://github.com/kubermatic/kubermatic/issues/5262) ([xrstf](https://github.com/xrs
+tf))
+- ACTION REQUIRED: Addon manifest templating is now a stable API, but different to the old implicit data. Custom addons might need to be adjusted. [#5275](https://github.com/kubermatic/kubermatic/issues/5275) ([xrstf](https://github.com/xrstf))
+- Added Flatcar Linux as an Operating System option
+- Add SLES as an Operating System option [#5040](https://github.com/kubermatic/kubermatic/issues/5040) ([kgroschoff](https://github.com/kgroschoff))
```
Comment (xrstf) [inline CHANGELOG.md]: It's a PITA, but can we normalize to present or past time (added vs. add)?

**Review by xrstf — CHANGES_REQUESTED** [review]
Reviewed.

File: CHANGELOG.md (hunk 1 above)
Comment (kdomanski) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 2 above)
Comment (kdomanski) [reply CHANGELOG.md]: I decided it makes more sense to remove the first line entirely and add the missing "supporter versions" section.

File: CHANGELOG.md (hunk 3 above)
Comment (kdomanski) [reply CHANGELOG.md]: You're totally right, I normally do that but I missed these.

**Review by xrstf — APPROVED** [review]
I like.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,125 @@
...
+- Fixed filter in combo dropdown [#2238](https://github.com/kubermatic/dashboard-v2/issues/2238) ([kgroschoff](https://github.com/kgroschoff))
+- Fixed node data dialog for vSphere clusters. [#2251](https://github.com/kubermatic/dashboard-v2/issues/2251) ([maciaszczykm](https://github.com/maciaszczykm))
+- Cluster creation time is now visible in the UI. [#2253](https://github.com/kubermatic/dashboard-v2/issues/2253) ([maciaszczykm](https://github.com/maciaszczykm))
+- Added info about End-of-life of Container Linux [#2264](https://github.com/kubermatic/dashboard-v2/issues/2264) ([kgroschoff](https://github.com/kgroschoff))
+- Enforcing pod security policy by the datacenter is now allowed. [#2270](https://github.com/kubermatic/dashboard-v2/issues/2270) ([maciaszczykm](https://github.com/maciaszczykm))
+- Introduced a number of responsiveness fixes to improve user experience on the smaller screens. [#2279](https://github.com/kubermatic/dashboard-v2/issues/2279) ([maciaszczykm](https://github.com/maciaszczykm))
+- Added support for system-wide dark/light theme selection. [#2294](https://github.com/kubermatic/dashboard-v2/issues/2294) ([floreks](https://github.com/floreks))
```
Comment (maciaszczykm) [inline CHANGELOG.md]: It's a duplicate of the first point in this section.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,125 @@
...
+- Added cluster metrics. [#1940](https://github.com/kubermatic/dashboard-v2/issues/1940) ([maciaszczykm](https://github.com/maciaszczykm))
+- Increased cpu &amp; memory defaults on vSphere [#1952](https://github.com/kubermatic/dashboard-v2/issues/1952) ([kgroschoff](https://github.com/kgroschoff))
+- Custom Presets are filtered by datacenter now [#1955](https://github.com/kubermatic/dashboard-v2/issues/1955) ([kgroschoff](https://github.com/kgroschoff))
+- Added notification panel. [#1957](https://github.com/kubermatic/dashboard-v2/issues/1957) ([maciaszczykm](https://github.com/maciaszczykm))
+- Added Pod Node Selector field. [#1968](https://github.com/kubermatic/dashboard-v2/issues/1968) ([maciaszczykm](https://github.com/maciaszczykm))
+- Operation Systems on VSphere for which no template is specified in datacenters are now hidden. [#1981](https://github.com/kubermatic/dashboard-v2/issues/1981) ([kgroschoff](https://github.com/kgroschoff))
+- Added metrics for nodes. [#1982](https://github.com/kubermatic/dashboard-v2/issues/1982) ([maciaszczykm](https://github.com/maciaszczykm))
```
Comment (zreigz) [inline CHANGELOG.md]: now we lost this feature :(

File: CHANGELOG.md (hunk 4 above)
Comment (maciaszczykm) [reply CHANGELOG.md]: This pull request is an enhacement of the first one. Right now we have dark, light and system preffered options in the theme selector.

cc @floreks

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,125 @@
...
+- Added possibility to specify customImage for GCP node deployments (required for RHEL). [#2190](https://github.com/kubermatic/dashboard-v2/issues/2190) ([maciaszczykm](https://github.com/maciaszczykm))
+- Fixed user settings layout on the smaller screens. [#2209](https://github.com/kubermatic/dashboard-v2/issues/2209) ([maciaszczykm](https://github.com/maciaszczykm))
+- Fixed loading Openstack flavors in add/edit node deployment dialog [#2222](https://github.com/kubermatic/dashboard-v2/issues/2222) ([floreks](https://github.com/floreks))
+- Fixed filter in combo dropdown [#2238](https://github.com/kubermatic/dashboard-v2/issues/2238) ([kgroschoff](https://github.com/kgroschoff))
+- Fixed node data dialog for vSphere clusters. [#2251](https://github.com/kubermatic/dashboard-v2/issues/2251) ([maciaszczykm](https://github.com/maciaszczykm))
+- Cluster creation time is now visible in the UI. [#2253](https://github.com/kubermatic/dashboard-v2/issues/2253) ([maciaszczykm](https://github.com/maciaszczykm))
+- Added info about End-of-life of Container Linux [#2264](https://github.com/kubermatic/dashboard-v2/issues/2264) ([kgroschoff](https://github.com/kgroschoff))
```
Comment (maciaszczykm) [inline CHANGELOG.md]: s/End-of-life/end-of-life?

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,125 @@
...
+- Added Pod Node Selector field. [#1968](https://github.com/kubermatic/dashboard-v2/issues/1968) ([maciaszczykm](https://github.com/maciaszczykm))
+- Operation Systems on VSphere for which no template is specified in datacenters are now hidden. [#1981](https://github.com/kubermatic/dashboard-v2/issues/1981) ([kgroschoff](https://github.com/kgroschoff))
+- Added metrics for nodes. [#1982](https://github.com/kubermatic/dashboard-v2/issues/1982) ([maciaszczykm](https://github.com/maciaszczykm))
+- Fixes issue that prevented creating Addons which had no AddonConfig deployed. [#1985](https://github.com/kubermatic/dashboard-v2/issues/1985) ([maciaszczykm](https://github.com/maciaszczykm))
+- Added possibility to collapse the sidenav. [#2004](https://github.com/kubermatic/dashboard-v2/issues/2004) ([kgroschoff](https://github.com/kgroschoff))
+- We now use WebSocket to get global settings. [#2008](https://github.com/kubermatic/dashboard-v2/issues/2008) ([maciaszczykm](https://github.com/maciaszczykm))
+- We now set &#39;SameSite=Strict&#39; for all cookies. [#2019](https://github.com/kubermatic/dashboard-v2/issues/2019) ([maciaszczykm](https://github.com/maciaszczykm))
```
Comment (maciaszczykm) [inline CHANGELOG.md]: Please remove this one as the next entry has overwritten it.

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,125 @@
...
+- Updated Thanos to v0.11.0 [#5176](https://github.com/kubermatic/kubermatic/issues/5176) ([youssefazrak](https://github.com/youssefazrak))
+- Updated Velero to v1.3.2 [#5326](https://github.com/kubermatic/kubermatic/issues/5326) ([scheeles](https://github.com/scheeles))
+
+**Dashboard**:
+
+- Added a dark theme and a selector to the user settings. [#1867](https://github.com/kubermatic/dashboard-v2/issues/1867) ([maciaszczykm](https://github.com/maciaszczykm))
+- Added possibility to define a default project in user settings. When a default project is choosen, the user will be automatically redirected to this project after login. Attention: One initial log in might be needed for the feature to take effect. [#1895](https://github.com/kubermatic/dashboard-v2/issues/1895) ([kgroschoff](https://github.com/kgroschoff))
+- Added UI support for dynamic kubelet config option [#1923](https://github.com/kubermatic/dashboard-v2/issues/1923) ([floreks](https://github.com/floreks))
+- Added Paginators to all tables [#1932](https://github.com/kubermatic/dashboard-v2/issues/1932) ([kgroschoff](https://github.com/kgroschoff))
```
Comment (maciaszczykm) [inline CHANGELOG.md]: s/Paginators/paginators

File: CHANGELOG.md (hunk 4 above)
Comment (kdomanski) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 5 above)
Comment (kdomanski) [reply CHANGELOG.md]: removed

File: CHANGELOG.md (hunk 6 above)
Comment (kdomanski) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 7 above)
Comment (kdomanski) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 8 above)
Comment (kdomanski) [reply CHANGELOG.md]: fixed

**Comment by maciaszczykm** [comment]
LGTM

**Comment by maciaszczykm** [comment]
@zreigz Can you take a final look and /lgtm?

**Pull request merged**